### PR TITLE
add int-to-separator-string method

### PR DESCRIPTION
**Pull request by nanaten**
add `toStringWithSeparated` method.
This method returns a string of numbers separated by national delimiters.

### CODE REVIEW

File: yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt [hunk 1]
```diff
@@ -80,4 +83,14 @@ val Int.isEven: Boolean
...
+    get() = (this % 2 == 1)
+
+/**
+ * return to string with specified country's separators.
+ * if the value is 123456, and locale is jp, returns to "123,456"
+
+ * @param locale target country locale
+ * @return string with separators
+ */
+fun Int.toStringWithSeparator(locale: Locale): String =
```
Comment (okuzawats) [inline yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt:95]: I think that providing default parameter would be useful (maybe US?), what do you think?

Comment (nanaten) [reply yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt]: I think so too! fixed!

**Comment by nanaten** [comment]
conflicts 🤔

File: yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt (hunk 1 above)
Comment (okuzawats) [reply yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt]: You should add a test case for default locale :)

Comment (nanaten) [reply yakan/src/main/java/com/okuzawats/yakan/IntExtensions.kt]: default locale test added!

**Comment by okuzawats** [comment]
Looks good to me except the difference of `.idea/misc.xml` is included.
It's my fault that it is old, but I think the difference should be removed from this PR :)

**Comment by nanaten** [comment]
@okuzawats Sorry, I have restored.

**Review by okuzawats — APPROVED** [review]
LGTM👍

**Pull request merged**